x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/migrations.ts

```diff
@@ -481,19 +481,19 @@ export default function createGetTests({ getService }: FtrProviderContext) {
     describe('8.7.0', () => {
       before(async () => {
         await kibanaServer.importExport.load(
-          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.5.0/cases_severity.json'
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.5.0/cases_severity_and_status.json'
```

Comment:
I noticed that the first case does not have a status. Is this by purpose so we can verify the correctness of the migration algorithm?

Reply:
That is exactly it 👍